boa_cli/src/debug/optimizer.rs

```diff
@@ -43,6 +43,39 @@ fn set_statistics(_: &JsValue, args: &[JsValue], context: &mut Context<'_>) -> J
     Ok(JsValue::undefined())
 }
 
+fn cfg(_: &JsValue, args: &[JsValue], _context: &mut Context<'_>) -> JsResult<JsValue> {
```

Comment:
Not sure if I would call this "cfg", since that can be confused with "config", and it's also the macro name used in Rust.

Maybe "cflowg", or something that shows that this is about "control flow graphs".